GliaWidgets/Sources/Theme/Theme+GVA.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+extension Theme {
+    var gvaStyle: GvaStyle {
+        let font = ThemeFontStyle.default.font
+
+        let persistentButton: GvaPersistentButtonStyle = .init(
+            title: .init(
+                textFont: font.bodyText,
+                textColor: .black,
+                backgroundColor: .clear
+            ),
+            backgroundColor: .fill(color: color.lightGrey),
+            cornerRadius: 8.49,
```

Comment:
I will ask about it from Aksel

Reply:
<img width="425" alt="Screenshot 2023-07-20 at 10 45 16" src="https://github.com/salemove/ios-sdk-widgets/assets/70878708/c6f4adef-5fa1-4915-8694-18d8878289a3">
<img width="443" alt="Screenshot 2023-07-20 at 10 49 37" src="https://github.com/salemove/ios-sdk-widgets/assets/70878708/21ae477f-de2e-41f3-91cb-227489ef21ca">


If I'm not mistaken, cornerRadius for PB is 5, and radius for the card is 8

Reply:
<img width="882" alt="Screenshot 2023-07-20 at 11 06 34" src="https://github.com/salemove/ios-sdk-widgets/assets/111733845/0e7390a4-579f-4310-b383-cf81cf3fc6ce">
<img width="915" alt="Screenshot 2023-07-20 at 11 07 03" src="https://github.com/salemove/ios-sdk-widgets/assets/111733845/6b5173b1-f7a3-40da-99fc-3e127288e842">

I honestly don't know, because I was relying on this section instead

Reply:
I would not suggest to rely on Code section at all) 
Usually it suggests useless awful code)

Reply:
Temporary fix is 10px as disgussed with Yaro. Changed in [2596c15](https://github.com/salemove/ios-sdk-widgets/pull/687/commits/2596c152c30b04a04dadd9660d6380611119e92a)